noxfile.py

```diff
@@ -32,7 +32,7 @@
 SKIP_CORE_TESTS = "0"
 SKIP_CORE_TESTS = os.environ.get("SKIP_CORE_TESTS", SKIP_CORE_TESTS) != "0"
 FIX = os.environ.get("FIX", "0") == "1"
-VERBOSE = os.environ.get("VERBOSE", "0")
+VERBOSE = os.environ.get("VERBOSE", "1")
```

Comment:
can you just limit this to the place where you want more output, specifically to when we call pytest instead of changing it for everything?

Reply:
sounds good!

Reply:
Also : even if we decided that we want to make VERBOSE true for CI (and I don't think this is what we are talking about), you should do with with the environment variable in the CircleCI config and not by changing the default like this.